tfhe/docs/fine_grained_api/Boolean/readme.md

```diff
@@ -104,7 +104,8 @@ fn main() {
 
 ## Encrypting inputs using a public key
 
-Once the server key is available on the **server side**, it is possible to perform some homomorphic computations. The client simply needs to encrypt some data and send it to the server. Again, the `Ciphertext` type implements the `Serialize` and the `Deserialize` traits, so that any serializer and communication tool suiting your use case can be utilized:
+Anyone (the server or a third party) with the public key can also encrypt some (or all) of the inputs.
+The public key can only be used to encrypt, not to decrypt.
```

Comment:
was this a bad copy paste, i.e. this sentence is present in the file elsewhere ?

Reply:
Yes, it seems so, the removed sentence is almost identical to the one in the previous paragraph

Reply:
perfect thanks